brainzutils/musicbrainz_db/tests/test_artist.py

```diff
@@ -1,26 +1,21 @@
-from unittest import TestCase
-from mock import MagicMock
+import pytest
+
 from brainzutils.musicbrainz_db.test_data import artist_linkin_park, artist_jay_z
 from brainzutils.musicbrainz_db.unknown_entities import unknown_artist
 from brainzutils.musicbrainz_db import artist as mb_artist
 
 
-class ArtistTestCase(TestCase):
-
-    def setUp(self):
-        mb_artist.mb_session = MagicMock()
-        self.mock_db = mb_artist.mb_session.return_value.__enter__.return_value
-        self.artist_query = self.mock_db.query.return_value.options.return_value.filter.return_value.all
+@pytest.mark.database
```

Comment:
in pytest, a _mark_ allows you to tag a particular test, and ask it to only run specific marks, or to exclude them. By marking db tests, we can run two separate sets of tests, the regular unit tests which run quickly, and the database ones, which might be slower